packages/babel/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@rollup/plugin-babel",
+  "version": "5.0.0-alpha.1",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Seamless integration between Rollup and Babel.",
+  "license": "MIT",
+  "repository": "rollup/plugins",
+  "author": "Rich Harris",
+  "homepage": "https://github.com/rollup/plugins/tree/master/packages/babel",
+  "bugs": "https://github.com/rollup/plugins/issues",
+  "main": "./dist/rollup-plugin-babel.cjs.js",
+  "engines": {
+    "node": ">= 8.0.0"
+  },
+  "scripts": {
+    "build": "rollup -c",
+    "prebuild": "rimraf dist",
+    "prepare": "pnpm run build",
+    "pretest": "pnpm run build",
+    "preversion": "pnpm run test",
+    "test": "ava"
+  },
+  "files": [
+    "dist",
+    "src"
+  ],
+  "keywords": [
+    "rollup-plugin",
+    "babel",
+    "es2015",
+    "es6"
+  ],
+  "peerDependencies": {
+    "@babel/core": "^7.0.0",
+    "rollup": "^1.20.0"
+  },
+  "dependencies": {
+    "@babel/helper-module-imports": "^7.7.4",
+    "@rollup/pluginutils": "^3.0.1"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.7.4",
+    "@babel/plugin-external-helpers": "^7.7.4",
+    "@babel/plugin-proposal-decorators": "^7.7.4",
+    "@babel/plugin-syntax-dynamic-import": "^7.7.4",
+    "@babel/plugin-transform-runtime": "^7.7.4",
+    "@babel/preset-env": "^7.7.4",
+    "@rollup/plugin-json": "^4.0.0",
+    "ava": "^2.4.0",
```

Comment:
`ava` and `rimraf` should be removed. we don't use `rimraf` in this repo, we have `del-cli` as a root dep, and ava is hoisted as well so no need to have it in devDeps. comparing against any of the other plugins' package.json is a very easy, fast way to get this right. any diff tool will work.